x-pack/plugins/infra/public/components/waffle/node.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiToolTip } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { InfraWaffleMapNode, InfraWaffleOptions } from '../../lib/lib';
+import { darken } from './lib/darken';
+
+interface Props {
+  onDrilldown: () => void;
+  squareSize: number;
+  options: InfraWaffleOptions;
+  node: InfraWaffleMapNode;
+  formatter: (val: number) => string;
+}
+
+export const Node: React.SFC<Props> = ({ node, formatter, squareSize }) => {
+  const metric = node.metrics.find(m => m.name === 'count');
+  const valueMode = squareSize > 75;
+  const label = 'Count';
+  const value = metric != null ? formatter(metric.value) : 'n/a';
+  return (
+    <EuiToolTip position="top" content={`${node.name} | ${value}`}>
+      <Container style={{ width: squareSize, height: squareSize }}>
+        <SquareOuter>
+          <SquareInner>
+            {valueMode && (
+              <ValueInner>
+                <Label>{label}</Label>
+                <Value>{value}</Value>
+              </ValueInner>
+            )}
+          </SquareInner>
+        </SquareOuter>
+      </Container>
+    </EuiToolTip>
+  );
+};
+
+export const Container = styled.div`
+  position: relative;
+  display: flex;
```

Comment:
It looks like this only contains absolutely positioned children, so why should this be a flex container?